package.json

```diff
@@ -2,7 +2,7 @@
   "author": "Jason Reynolds",
   "name": "discourse-api",
   "description": "A simple Node wrapper for the Discourse API",
-  "version": "1.8.0",
+  "version": "1.10.0",
```

Comment:
Version management would generally be done by the maintainer of the project once they had assessed the affect of the changes on the existing API and applied semver rules to it. I suspect that the JSON output fix would be a breaking change to the API response, so this might warrant a major version bump to prevent existing users suddenly breaking as they were likely having to `JSON.parse` the responses before.